pkgs/applications/editors/marker/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+, gtk3
+, gtksourceview
+, gtkspell3
+, webkitgtk
+, pandoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "marker";
+  version = "2020.04.04";
+
+  src = fetchFromGitHub {
+    owner = "fabiocolacio";
+    repo = "Marker";
+    rev = "${version}";
+    fetchSubmodules = true;
```

Comment:
It looks like this is vendoring https://github.com/Mandarancio/scidown. Is marker able to use a system installation of scidown? I think that would be preferable

Reply:
Thank you for your review! Marker's `meson.build` is compiling it all together by explicitly listing all the source files in scidown as opposed to using scidown's build system and then linking after the fact. If I packaged scidown, we would need a pretty big patch for the `meson.build` file. But if you think it is worth it, I can look into it when I get a chance.

Reply:
Ah you're right, didn't realize those are all hardcoded. Typically I'd expect a submodule to have a fallback to an external library

Reply:
Will keep as is then. Yes, you are right that it in general it would be better from a dependency management perspective to have this separate project in Nix as its own package. As you point out, the hardcoded build using the submodules sources is one reason for keeping it as is, and the other is that I do not think scidown is used in many other projects, but who knows, maybe it will be in the future (though there has not been much recent activity in the repository).